docs/worker-node/using-wn.md

```diff
@@ -10,6 +9,16 @@ The OSG provides no scientific software dependencies or software build tools on
 
 If you would like to test the minimal OS environment that jobs can expect, you can test out your scientific software in [the OSG Docker image](https://hub.docker.com/r/opensciencegrid/osg-wn/).
 
+Hardware Recommendations
+------------------------
+| Hardware               | Minimum | Recommended                         | Notes                                             |
+|:-----------------------|:--------|:----------------------|:--------------------------------------------------|
+|Core per pilot                   |  1      |8                      | Dependends on the supported VOs. The total core count on every node in the cluster must be divisible by core per pilot.|
+|Memory per core                 | 1024MB  | 2048MB                  | Memory per core times core per pilot needs to be less than the total memory on every node. Do not overcommit. |
+|Scratch disk per core ([OSG_WN_TMP](/worker-node/using-wn/#osg_wn_tmp))| 2GB    | 10 GB                  | This can be overcommitted if a mix of different VO jobs is expected.|
```

Comment:
Minor style suggestion

```suggestion
|Scratch disk per core ([OSG_WN_TMP](#osg_wn_tmp))| 2GB    | 10 GB                  | This can be overcommitted if a mix of different VO jobs is expected.|
```